infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesClientFactory.java

```diff
@@ -147,8 +147,11 @@ protected OkHttpClient getHttpClient() {
    *     infromation
    */
   public OkHttpClient getAuthenticatedHttpClient() throws InfrastructureException {
-    throw new InfrastructureException(
-        "Impersonating the current user is not supported in the Kubernetes Client.");
+    /*
```

Comment:
A small nitpick - this comment might be better placed in the javadoc of the method, because it explaines why we violate the expectations set out in the method's javadoc.